lib/loadtest.js

```diff
@@ -453,6 +454,7 @@ var Operation = function(options, Constructor, callback)
 			self.latency.show();
 		}
 	}
+	this.stop = stop;
```

Comment:
Instead of redefining `this.stop`, I would prefer if you directly defined the function as `self.stop = function(){...}`.